### PR TITLE
[TW-1257] Doc performance testing improvements

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Add error debugging section for viewing top 10 requests per error class
- Note you can run a folder as well as a collection
- Note you can use collections linked to APIs for manual collection runs, CLI runs, and performance tests

View the beta docs here: https://tw-1257-performance-testing-updates.learning.postman-beta.com/docs/collections/testing-api-performance/

### CODE REVIEW

File: src/pages/docs/collections/testing-api-performance.md [hunk 1]
```diff
@@ -176,6 +180,16 @@ From the **Errors** tab, you can take the following actions:
 
 <img alt="Viewing top errors" src="https://assets.postman.com/postman-docs/v10/performance-test-filter-errors-v10-15.jpg" width="600px"/>
 
+### Debugging errors
+
+You can view details about errors that occurred during the performance test to help in the debugging process. On the **Errors** tab, expand an error class below the graph. Postman displays the 10 most frequent requests for each error class and the number of times each request encountered that error.
+
+Select a request to view details about the request, including the response, headers, and request body. Use the controls on the **Response** tab to format, copy, or search within the response. Select the name of a request to open the request.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/testing-api-performance.md:187]:
```suggestion
Select a request to view details about it, including the response, headers, and request body. Use the controls on the **Response** tab to format, copy, or search within the response. Select the name of a request to open it.
```

Comment (MatthewPinsker-postman) [reply src/pages/docs/collections/testing-api-performance.md:187]: Minor suggestion

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/collections/testing-api-performance.md [hunk 2]
```diff
@@ -176,6 +180,16 @@ From the **Errors** tab, you can take the following actions:
...
 
+### Debugging errors
+
+You can view details about errors that occurred during the performance test to help in the debugging process. On the **Errors** tab, expand an error class below the graph. Postman displays the 10 most frequent requests for each error class and the number of times each request encountered that error.
+
+Select a request to view details about it, including the response, headers, and request body. Use the controls on the **Response** tab to format, copy, or search within the response. Select the name of a request to open it.
+
+<img alt="Viewing error details" src="https://assets.postman.com/postman-docs/v10/performance-test-errors-detail-v10-17.jpg"/>
+
+> Error class data is persisted locally during your current Postman session and isn't saved permanently. If you close the current tab or quit Postman, the data is removed.
```
Comment (malvikach) [inline src/pages/docs/collections/testing-api-performance.md:191]: We can say Request/Response details are not available. Error class and which requests (plus frequency) is still available, see below:
check slack: https://postman.slack.com/archives/C04NJAHK1UG/p1691073937903839

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/collections/testing-api-performance.md [hunk 3]
```diff
@@ -176,6 +180,16 @@ From the **Errors** tab, you can take the following actions:
 
 <img alt="Viewing top errors" src="https://assets.postman.com/postman-docs/v10/performance-test-filter-errors-v10-15.jpg" width="600px"/>
 
+### Debugging errors
+
+You can view details about errors that occurred during the performance test to help in the debugging process. On the **Errors** tab, expand an error class below the graph. Postman displays the 10 most frequent requests for each error class and the number of times each request encountered that error.
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/collections/testing-api-performance.md]: I updated the "Debugging errors" section with this information

File: src/pages/docs/collections/testing-api-performance.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/collections/testing-api-performance.md]: I updated the note with this information

**Review by postfrzero — APPROVED** [review]
LGTM!

**Pull request merged**